src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -2020,10 +1989,10 @@ bool AdaptDispatch::_ModeParamsHelper(const DispatchTypes::ModeParams param, con
         // It also makes more sense to not bubble it up, because this mode is specifically for INPUT_RECORD interop
         // and thus entirely between a PTY's input records and its INPUT_RECORD-aware VT-aware console clients.
         // Returning true here will mark this as being handled and avoid this.
-        return true;
+        break;
     default:
         // If no functions to call, overall dispatch was a failure.
-        return false;
+        break;
```

Comment:
This comment seems a bit pointless now without the `return false`. I'd be inclined to nuke it.